src/components/Articles/Feed.jsx

```diff
@@ -190,6 +210,16 @@ export class Feed extends Component {
               </div>
             </div>
           </div>
+          <div className="pagination">
+            <button type="button" className="paginationBtnsIcons" onClick={this.changePaginationLeft}><img src="https://image.flaticon.com/icons/svg/892/892513.svg" alt="" className="paginationIconBack" /></button>
+            <button type="button" value="1" onClick={this.changePage} className="paginationBtns" id="btn1">1</button>
```

Comment:
you hardcoded the buttons from `1 to 2` you can make this a 4 liner or 6, kindly revise this and implement it.

Reply:
Give me an example implementation of this

Reply:
```javascript
for(let i = 1; i <= 6; i++) <button type="button" value=`${i}` onClick={this.changePage} className="paginationBtns" id=`btn${i}`>i</button>
```